docker-compose.yaml

```diff
@@ -467,7 +467,7 @@ services:
 
   zac:
     # By default, we use the latest ZAC Docker Image. Change this if you wish to use a specific version.
-    image: ${ZAC_DOCKER_IMAGE:-ghcr.io/infonl/zaakafhandelcomponent:latest}
+    image: ${ZAC_DOCKER_IMAGE:-ghcr.io/infonl/zaakafhandelcomponent:2.6.1}
```

Comment:
This doesn't look like it belongs in this PR

Reply:
This PR is no longer relevant. We switched to Renovate now.